airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -159,64 +159,56 @@ private void doScanLibrary() {
         LOG.info("Starting to scan media library.");
         Date lastScanned = DateUtils.truncate(new Date(), Calendar.SECOND);
         LOG.debug("New last scan date is " + lastScanned);
+        // Maps from artist name to album count.
+        Map<String, Integer> albumCount = new HashMap<String, Integer>();
+        Genres genres = new Genres();
 
-        try {
```

Comment:
I don't think this `try` .. `catch` is useless.  It makes sure that the finalization code (`finally` block) gets run even if an exception occurs during the processing.